index.css

```diff
@@ -239,6 +239,101 @@ body {
   color: #172b4d;
 }
 
+.poptitle {
+  font-weight: bold;
+  font-size: x-large;
+  color: #172b4d;
+}
+
+.pop-up-menu {
+  width: 90vw;
+  height: 90vh;
+  background: white;
+  display: flex;
+  justify-content: center;
+  margin-top: 5px;
+}
+
+.pop-up-menu-wrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
Great design team 🤝

- [ ] As stated in the project requirements, kindly ensure your design replicates the Figma design as much as possible. Kindly ensure the buttons are well positioned on the page and they are not overflowing.
- Your design 
![your-mobile-popup](https://github.com/Samuelwanza/portfolio_setup_and_mobile_first/assets/63917725/9eb76b31-21b3-4fa5-b6b4-a16d2b9aaef6)
- Given design
![Given-mobile-popup](https://github.com/Samuelwanza/portfolio_setup_and_mobile_first/assets/63917725/0feed33f-7668-4fc3-a894-1219b6a45ac0)